terratest/run-tests-from-cmdline.md

```diff
@@ -6,6 +6,7 @@ The test can also be run from the command line which allows you to avoid complyi
 - You have a local clone of this repo
 - You have installed an appropriate go version (v1.13 or later)
 - You have installed dep
+- You've disabled go modules by setting the environment variable `GO111MODULE="off"` (v1.16 or later)
```

Comment:
keep spelling out the "you have" - why break the model on the fourth try?? ;-)
You have disabled go modules by setting the environment variable `GO111MODULE="off"` (v1.16 or later)